quic_lb.c

```diff
@@ -400,3 +499,49 @@ quic_lb_decrypt_cid(void *ctx, void *cid, void *sid, size_t *cid_len)
             context->decrypt(ctx, cid, sid, cid_len));
 }
 
+void
```

Comment:
Is this being exercised somehow?

Reply:
It was - it seems it got lost in transit. Addressing :)